app/src/main/java/com/sdp13epfl2021/projmag/activities/SignInActivity.kt

```diff
@@ -91,6 +90,9 @@ class SignInActivity : AppCompatActivity() {
                             startActivity(intent)
                             finish()
                         } else if(isNew == false) {
+                            val savedTypeChoice :String = "userTypeChoicePrefs"
+                            val settings = getSharedPreferences(savedTypeChoice, 0)
```

Comment:
Using a magic number (0) is more difficult to understand than this.
```suggestion
                            val settings = getSharedPreferences(savedTypeChoice, Context.MODE_PRIVATE)
```